conjurapi/authn/auth_token.go

```diff
@@ -11,27 +11,10 @@ const (
 	TimeFormatToken4 = "2006-01-02 15:04:05 MST"
 )
 
-type AuthnToken interface {
```

Comment:
Interface is no longer needed since we're not doing any polymorphism w/ v4 vs v5 tokens.